esrally/telemetry.py

```diff
@@ -1408,6 +1409,194 @@ def record(self):
             self.metrics_store.put_doc(doc, level=MetaInfoScope.cluster, meta_data=data_stream_metadata)
 
 
+class IngestPipelineStats(TelemetryDevice):
+    internal = False
+    command = "ingest-pipeline-stats"
+    human_name = "Ingest Pipeline Stats"
+    help = "Reports Ingest Pipeline stats at the end of the benchmark."
+
+    def __init__(self, clients, metrics_store):
+        super().__init__()
+        self.logger = logging.getLogger(__name__)
+        self.clients = clients
+        self.metrics_store = metrics_store
+        self.start_stats = {}
+        self.specified_cluster_names = self.clients.keys()
+
+        self.ingest_pipeline_cluster_count = 0
+        self.ingest_pipeline_cluster_time = 0
+        self.ingest_pipeline_cluster_failed = 0
+
+    def on_benchmark_start(self):
+        self.logger.debug("Gathering Ingest Pipeline stats at benchmark start")
```

Comment:
looks like there's a mix for log level with various devices, but this seems `info` to me